<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-LMKUKFJ" failures="0" tests="9" name="freedodomain.appium_3" time="0.000" errors="1" timestamp="2023-11-16T21:09:11 IST" skipped="8">
  <testcase classname="freedodomain.appium_3" name="TC_0060_Admin_approved_request" time="0.000">
    <skipped/>
  </testcase> <!-- TC_0060_Admin_approved_request -->
  <system-out/>
  <testcase classname="freedodomain.appium_3" name="TC_0062_Verify_user_End_ride_request" time="0.000">
    <skipped/>
  </testcase> <!-- TC_0062_Verify_user_End_ride_request -->
  <system-out/>
  <testcase classname="freedodomain.appium_3" name="logger" time="0.000">
    <error message="Cannot invoke &quot;org.apache.logging.log4j.Logger.info(String)&quot; because &quot;com.utility.BaseClass.log&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.logging.log4j.Logger.info(String)" because "com.utility.BaseClass.log" is null
at com.utility.BaseClass.logger(BaseClass.java:233)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- logger -->
  <system-out/>
  <testcase classname="freedodomain.appium_3" name="TC_0063_Verify_user_end_ride_booking_status" time="0.000">
    <skipped/>
  </testcase> <!-- TC_0063_Verify_user_end_ride_booking_status -->
  <system-out/>
  <testcase classname="freedodomain.appium_3" name="TC_0061_Verify_user_able_to_view_booking_status_all_elements" time="0.000">
    <skipped/>
  </testcase> <!-- TC_0061_Verify_user_able_to_view_booking_status_all_elements -->
  <system-out/>
  <testcase classname="freedodomain.appium_3" name="TC_0064_Admin_check_End_ride_request" time="0.000">
    <skipped/>
  </testcase> <!-- TC_0064_Admin_check_End_ride_request -->
  <system-out/>
  <testcase classname="freedodomain.appium_3" name="TC_0059_Verify_user_able_to_Online_payment_booking" time="0.000">
    <skipped/>
  </testcase> <!-- TC_0059_Verify_user_able_to_Online_payment_booking -->
  <system-out/>
  <testcase classname="freedodomain.appium_3" name="TC_0065_Verify_user_Complete_ride_booking_status" time="0.000">
    <skipped/>
  </testcase> <!-- TC_0065_Verify_user_Complete_ride_booking_status -->
  <system-out/>
  <testcase classname="freedodomain.appium_3" name="TC_0066_Verify_after_Complete_ride_invoice_button" time="0.000">
    <skipped/>
  </testcase> <!-- TC_0066_Verify_after_Complete_ride_invoice_button -->
  <system-out/>
</testsuite> <!-- freedodomain.appium_3 -->
